app/representers/account_representer.rb

```diff
@@ -54,6 +55,8 @@ module AccountRepresenter
   end
 
   property :state
+  property :bought_cinstances_count
+  property :admin_name
```

Comment:
These 2 need to be tested in:
https://github.com/3scale/porta/blob/23c541044bf72ea2e9632129877ef0d393e9e32c/spec/acceptance/api/account_spec.rb#L5